lib/framework/stub/camera_manager.dart

```diff
@@ -4,11 +4,18 @@ import 'package:ensemble/framework/scope.dart';
 import 'package:flutter/cupertino.dart';
 
 abstract class CameraManager {
+  Future<bool?> hasPermission();
   Future<void> openCamera(BuildContext context, ShowCameraAction cameraAction,
       ScopeManager? scopeManager);
 }
 
 class CameraManagerStub extends CameraManager {
+  @override
+  Future<bool?> hasPermission() {
+    throw ConfigError(
```

Comment:
We don't throw an Error here, simply false. This is valid use case. We only throw the error if the user attempts to use the camera.

Reply:
I updated it. Can you check now?